src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: boolean,
```

Comment:
Why do you need this variable? Try to rewrite without it.

Reply:
pressedKey is the task condition.
Make the App a class component with pressedKey in the state.

Reply:
Yes, but you could hold there pressedKey, not boolean.